tests/ceph_ansible/test_ansible_upgrade.py

```diff
@@ -146,6 +148,15 @@ def run(ceph_cluster, **kw):
     if config.get("ansi_cli_args"):
         cmd += config_dict_to_string(config["ansi_cli_args"])
 
+    if limit_node:
+        short_name_list = [
```

Comment:
I would suggest to use below definitions to get node(s). so we could re-use common code.
https://github.com/red-hat-storage/cephci/blob/master/ceph/utils.py#L978

Reply:
these methods would be helpful even if we have multiple node names.